pandas_log/pandas_execution_stats.py

```diff
@@ -85,27 +97,34 @@ def persist_execution_stats(self):
         set_df_attr(self.output_df, "execution_history", prev_exec_history)
         append_df_attr(self.output_df, "execution_history", self)
 
-    def log_stats_if_needed(self, silent, verbose):
+    def log_stats_if_needed(self, silent, verbose, copy_ok):
 
         from pandas_log.pandas_log import ALREADY_ENABLED
 
         if silent or not ALREADY_ENABLED:
             return
 
-        if (
-            verbose
-            or self.fn.__name__ not in PANDAS_ADDITIONAL_METHODS_TO_OVERIDE
-        ):
-            print(self)
+        if verbose or self.fn.__name__ not in DATAFRAME_ADDITIONAL_METHODS_TO_OVERIDE:
```

Comment:
@cmdavis4 I have run this and if you run pandas methods without the context manager aka with pandas_log.enable():

Reply:
not sure I follow

Reply:
@cmdavis4 the following code will now print stats and it shouldnt
import pandas_log
... <some random pandas method>

Reply:
I don't get the same behavior. You're saying if you run

```
import pandas as pd
import pandas_log
df = pd.DataFrame([[1,2,3],[4,5,6]])
df.head()
```

It will print the head log message?

Reply:
my bad :)